src/SoundList.js

```diff
@@ -9,7 +9,35 @@ function timeConvert(minutes) {
   return `${(min < 10 ? "0" : "") + min}:${secs < 10 ? "0" : ""}${secs}`;
 }
 
-const SoundList = ({ tracks, header, onSoundClick, className = "" }) => {
+// Psuedo code:
+// function highlightSound(track, sound) {
```

Comment:
We should probably remove this comment

Reply:
I'll remove the pseudocode.

Reply:
I can do it, I just need help with the questions I asked above first ^^^

Reply:
Okay. I'll leave the pseudocode. I responded to the questions you asked in another conversation.